service/controller/v24/cluster_resource_set.go

```diff
@@ -493,6 +494,20 @@ func NewClusterResourceSet(config ClusterResourceSetConfig) (*controller.Resourc
 		}
 	}
 
+	var stackOutputResource controller.Resource
+	{
+		c := stackoutput.Config{
+			Logger: config.Logger,
+
+			Route53Enabled: config.Route53Enabled,
+		}
+
+		stackOutputResource, err = stackoutput.New(c)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+	}
```

Comment:
The structure in this project is weird. We should have separate packages for `drainer` and `awsconfig` controller. I wonder where this resource should be if we had that.

Reply:
Probably in `pkg`.